### PR TITLE
Feature - Code Clean up and unit testing for the Standings Feature

### PR DESCRIPTION
**Pull request by ksookdhew**
## Description
Code Clean up and unit testing for the Standings Feature

## Checklist 
Please review and check the following before submitting this pull request: 
- [x] The code follows the project's coding standards. 
- [x] 100% Unit test coverage for View Models 
- [x] All warnings Resolved.

### CODE REVIEW

**Review by AShozi — APPROVED** [review]

File: F1/F1/Standings/DriverStandingTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//
+//  AllResultsTableViewCell.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/08.
+//
+
+import UIKit
+
+class DriverStandingTableViewCell: UITableViewCell {
```
Comment (kieran-woodrow) [inline F1/F1/Standings/DriverStandingTableViewCell.swift:10]: This looks similar to the other cell. In future, if you want to try, see if you can use the saem cell and make it generic to limit the same code being dome twice :)

File: F1/F1Tests/ConstructorStandingsViewModelTests.swift [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    func testEmptyDriver() {
+        let firstResult = viewModel.drivers(driversList: [nil, nil, nil])
+        XCTAssertEqual(firstResult, "/")
+
+        let secondResult = viewModel.drivers(driversList: [nil])
+        XCTAssertEqual(secondResult, "")
+    }
+}
```
Comment (kieran-woodrow) [inline F1/F1Tests/ConstructorStandingsViewModelTests.swift:100]: Nice job!

**Review by kieran-woodrow — APPROVED** [review]
very good job with the tests. Approving, but read through the comments i made along the way :)

File: F1/F1/Results/Repository/ResultsRepository.swift [hunk 3]
```diff
@@ -19,7 +19,7 @@ class ResultsRepository: ResultsRepositoryType {
         URLSession.shared.request(endpoint: url, method: .GET, completion: completion)
     }
     func fetchRacingResults(completion: @escaping (ResultsResults)) {
-        let url = "https://ergast.com/api/f1/current/results.JSON?limit=100"
+        let url = "https://ergast.com/api/f1/current/results.JSON?limit=200"
```
[the remark this answers is not in the document]
Comment (ksookdhew) [reply F1/F1/Results/Repository/ResultsRepository.swift:22]: The thing with my API is that it is nested. So it looks like i'm fetching 200 but I need the "7" outer objects to be displayed on the initial screen and then I use the inner objects on the next screen. There is no other endpoint that gives me just the outer objects either.

File: F1/F1/Standings/DriverStandingTableViewCell.swift (hunk 1 above)
Comment (ksookdhew) [reply F1/F1/Standings/DriverStandingTableViewCell.swift:10]: So I did consider this in the beginning but there are a few differences and I decided not the sake of the way I did it was quicker. I am definitely keeping this in mind for improvements and will implement after I have done everything else 😄

File: F1/F1/Standings/ConstructorStandings/ConstructorStandingsViewModel.swift [hunk 4]
```diff
@@ -18,19 +19,33 @@ class ConstructorStandingsViewModel {
...
+        constructorStanding?.count ?? 0
     }
 
+    // MARK: Functions
     func constructor(atIndex: Int) -> ConstructorStanding? {
-        return constructorStanding?[atIndex] ?? nil
+        constructorStanding?[atIndex] ?? nil
+    }
+
+    func drivers(driversList: [Driver?]) -> String {
```
[the remark this answers is not in the document]
Comment (ksookdhew) [reply F1/F1/Standings/ConstructorStandings/ConstructorStandingsViewModel.swift:32]: Works great! 😄

File: F1/F1/Results/Repository/ResultsRepository.swift (hunk 3 above)
[the remark this answers is not in the document]
Comment (kieran-woodrow) [reply F1/F1/Results/Repository/ResultsRepository.swift:22]: oh i see. Okay. But point is still there, if you can implement paging, try it out if you can. But not a must do, just if you want to go above and beyond :)

[the remark this answers is not in the document]
Comment (ksookdhew) [reply F1/F1/Results/Repository/ResultsRepository.swift:22]: Good point. Will definitely look more into it when im polishing the app 👍

File: F1/F1/Standings/ConstructorStandings/ConstructorStandingsViewModel.swift (hunk 4 above)
[the remark this answers is not in the document]
Comment (kieran-woodrow) [reply F1/F1/Standings/ConstructorStandings/ConstructorStandingsViewModel.swift:32]: cool!. If you find yourself doing if statements and more if statements, or something that seems like a bit of a shlep..Have a look at the higher order functions available to you to use like comactMp. They really help!
Go research compactMap, map, flatmap, contains, allsatisfy, split, etc :)

File: F1/F1/Standings/ConstructorStandingTableViewCell.swift [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    @IBOutlet weak private var position: UILabel!
+    @IBOutlet weak private var drivers: UILabel!
+    @IBOutlet weak private var name: UILabel!
+    @IBOutlet weak private var points: UILabel!
+    @IBOutlet weak private var constructorImage: UIImageView!
+
+    // MARK: Functions
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        contentView.layer.cornerRadius = 10
```
[the remark this answers is not in the document]
Comment (ksookdhew) [reply F1/F1/Standings/ConstructorStandingTableViewCell.swift:22]: I've added to list of improvements 👍

**Review by sjacobs1 — APPROVED** [review]

**Review by giacatano — APPROVED** [review]

**Pull request merged**